src/main/java/gov/usds/case_issues/services/UserService.java

```diff
@@ -0,0 +1,28 @@
+package gov.usds.case_issues.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import gov.usds.case_issues.db.model.OAuthUser;
+import gov.usds.case_issues.db.repositories.UserRepository;
+
+@Service
+@Transactional
```

Comment:
Should be `readOnly=false`—I think that's the default, but it's nice to be explicit. (Or, put that annotation on the method, and `readOnly=true` on the class-level annotation.)